### ISSUE TITLE
UnitControl: Unit selector is not tabbable

### ISSUE THREAD

**Issue by ItsJonQ** [WordPress/gutenberg#25703]
<img width="224" alt="Screen Shot 2020-09-29 at 10 13 57 AM" src="https://user-images.githubusercontent.com/2322354/94570129-7e963a80-023c-11eb-83ca-bae64751544a.png">

The unit select within `UnitControl` is not keyboard accessible.

For context: During the initial creation of this component, I was unsure whether it should be keyboard tabbable or not. The reason is because when they are used inline (like in a BoxControl for padding), one would have to double press tab to jump to the next value.

I wasn't sure what the best pattern was 🤔 .

I opted to leave it disabled, but configure the component (via a prop) to enable tab focus if we needed to (in the future).

**To reproduce**
1. Add a Cover block
2. Click on the height control
3. Press <kbd>Tab</kbd>
4. Cannot tab into unit select

**Editor version (please complete the following information):**
- Gutenberg `master` branch
